### ISSUE TITLE
Подробнее оп роцессах

### ISSUE THREAD

**Issue by rgordeev** [avito-tech/playbook#8]
Собственно, хочется подробного описания о том, как организованы процессы, а главное какие гарницы видите по применению подобных решений. Также хочется узнать, как продуктовнеры управляют продуктами, как таких людей готовите/находите, какой у вас опыт от формирования спринтов в зависимых проектах (как происходит синхронизация, как версионируете продукты и прочее). В общем, хочется детелей. Сейчас описание суховато и скудновато.

**Comment by etolstoy** [comment]
@rgordeev Привет!
Можешь, пожалуйста, уточнить, какие конкретно процессы ты имеешь в виду? Или речь как раз про управление продуктами, спринтами и наймом?

**Comment by rgordeev** [comment]
@etolstoy Да, именно о них. Хотелось бы потроблее узнать, как организуете спринты, как решаете проблемы синхронизации версий разных компонентов и прочие интересные моменты из жизни продуктовых команд.
